app/javascript/views/referrals/referralsCard/style.jsx

```diff
@@ -0,0 +1,162 @@
+import styled, { css } from 'styled-components';
+
+export const StyledWrapper = styled.div
+`
+  border-radius: 6px;
+  background-color: white;
+  box-shadow: rgba(99, 105, 110, 0.18) 0px 1px 12px 0px;
+  padding: 30px;
+`;
+
+export const StyledGrid = styled.div
```

Comment:
Do we have to name all of these `StyledX` I feel like we would know that it's styled based on the fact that we imported it from Styled?

Reply:
I took this convention from Brave-ui, not sure if they had a reason for this, but I'd be okay with dropping the Styled prefix

Reply:
I think it may be to prevent naming clash with Components 

for example if you have a component `<Card/>` and want to create a styled class called `Card` you may get an error

Although that's a lot of extra syntax for this benefit, I'm leaning towards removing the `Styled` prefix